plugins/woocommerce-admin/client/marketplace/components/product-card/product-card-footer.tsx

```diff
@@ -144,6 +144,32 @@ function ProductCardFooter( props: { product: Product } ) {
 		return '';
 	}
 
+	const getReaderPriceLabel = () => {
+		if ( product.isOnSale ) {
+			return sprintf(
+				//translators: %1$s is the sale price of the product, %2$s is the regular price of the product, %3$s is the billing period
+				__( 'Sale Price %1$s, %2$s %3$s', 'woocommerce' ),
```

Comment:
I found the announcement of the regular price confusing, might help to spell it out.

```suggestion
				__(
					'Sale Price %1$s  %3$s, regular price %2$s %3$s',
					'woocommerce'
				),
```

Reply:
done in [f121b87](https://github.com/woocommerce/woocommerce/pull/51499/commits/f121b8711ec22b1e7e680700186fdd6e0ab12576)